Sources/FileKit/FileKit.swift

```diff
@@ -17,51 +17,24 @@
 import Foundation
 import LoggerAPI
 
-/// Utility providing path resolution and execution environment information
+/// Resolves commonly used paths, including the project, executable and working directories.
 public class FileKit {
 
-    // MARK: Absolute Path Resolution
+    // MARK: Path to Executable Folder
 
-    /// Absolute path to the executable's folder
+    /// Absolute path to the executable's folder.
```

Comment:
Same comment as above

Reply:
I've clarified the wording, added examples and the README and apidoc are now in sync.